kf_model_fhir/mappers/resources/group.py

```diff
@@ -0,0 +1,69 @@
+"""
+This module converts Kids First families to FHIR Groups.
+"""
+import pandas as pd
+
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from common.utils import make_identifier, make_select, get
+
+RESOURCE_TYPE = 'Group'
+
+
+group_type = {
+    constants.SPECIES.DOG: 'animal',
+    constants.SPECIES.HUMAN: 'person'
+}
+
+
+def yield_groups(eng, table, study_id, kfdrc_patients):
+    df = pd.DataFrame([
+        dict(row) for row in make_select(
+            eng, table,
+            CONCEPT.FAMILY.ID,
+            CONCEPT.PARTICIPANT.SPECIES,
+            CONCEPT.PARTICIPANT.ID
+        )
+    ])
+    
+    for family_id, group in df.groupby(CONCEPT.FAMILY.ID):
+        if not family_id:
+            continue
+
+        retval = {
+            'resourceType': RESOURCE_TYPE,
+            'id': make_identifier(RESOURCE_TYPE, study_id, family_id),
+            'meta': {
+                'profile': [
+                    'http://hl7.org/fhir/StructureDefinition/Group'
+                ]
+            },
+            'identifier': [
+                {
+                    'system': f'https://kf-api-dataservice.kidsfirstdrc.org/families?study_id={study_id}',
```

Comment:
Should this be `'system': f'https://kf-api-dataservice.kidsfirstdrc.org/families?external_id=',` ?

Reply:
For other mappers like [this] (https://github.com/kids-first/kf-model-fhir/blob/master/kf_model_fhir/mappers/resources/kfdrc_patient.py#L142-L143), I made the system contextualized within a study under a certain endpoint. But, in the [new plugin](https://github.com/kids-first/kf-model-fhir/blob/ingest-plugin/kf_model_fhir/ingest_plugin/kfdrc_patient_no_phi.py#L139-L140), I figured you made it point to that specific entity. For the current mappers, I may use the previous approach for the consistency across all mappers and take the new approach for the plugin development?

Reply:
You're right, it should also include the study

Reply:
Updated!